src/cli/commands/licenses.js

```diff
@@ -124,15 +125,14 @@ export const {run, setFlags, examples} = buildSubCommands('licenses', {
       byLicense.set(manifest.name, manifest);
     }
 
-    console.log(
+    reporter.log(
       'THE FOLLOWING SETS FORTH ATTRIBUTION NOTICES FOR THIRD PARTY SOFTWARE THAT MAY BE CONTAINED ' +
-        `IN PORTIONS OF THE ${String(manifest.name).toUpperCase().replace(/-/g, ' ')} PRODUCT.`,
+        `IN PORTIONS OF THE ${String(manifest.name).toUpperCase().replace(/-/g, ' ')} PRODUCT.` +
+        '\n',
     );
-    console.log();
 
     for (const [licenseText, manifests] of manifestsByLicense) {
-      console.log('-----');
-      console.log();
+      reporter.log('-----\n');
```

Comment:
`reporter.log` dumps a bunch of ansi escapes to clear the current line and is for abstracting the console output so it can be consumed by other tools (json output being the primary one). This command is only for text consumption and you should just be dumping it to a txt file. Using a reporter here has the potential to mess up the output since it might print ansi escapes.